lib/middleware/profile.js

```diff
@@ -2,7 +2,13 @@ module.exports = settings => {
   const { Profile } = settings.models;
   return (req, res, next) => {
 
-    delete req.user.access_token;
+    // define access token as a non-enumerable property so it is accessible in code downstream
+    // but is not enumerated in activity logs event data
+    const token = req.user.access_token;
+    Object.defineProperty(req.user, 'access_token', {
+      enumerable: false,
+      value: token
+    });
```

Comment:
good plan

Reply:
There's probably large chunks of the data that ends up in activity logs that should get the same treatment tbh.